INSTRUCTION.md

```diff
@@ -0,0 +1,89 @@
+## Deploy the application following the commands:
+```
+cd ./.infrastructure/  
+kubectl apply -f namespace.yml
+kubectl apply -f busybox.yml
```

Comment:
There is a potential typo in the namespace used in the command. Ensure that the namespace 'todoapp' is correct and consistent with the rest of your configuration, as it should match the namespace defined in your Kubernetes configurations.